packages/@guardian/source-foundations/src/palette.ts

```diff
@@ -310,7 +310,7 @@ export const background = {
 	ctaTertiaryHover: '#E5E5E5',
 	input: neutral[100],
 	inputChecked: brand[500],
-};
+} as const;
```

Comment:
@SiAdcock  quick thought – I added `as const` here, because some of these colours don't pull from the `colors` object defined above. Is that alright, or would we ideally want them all to live in the `colors` object?

Reply:
The hardcoded colours here are snowflakes that don't live on the palette. Ideally these would be defined using some darkening algorithm e.g.  `ctaSecondaryHover: darken(ctaSecondary, '10%')` 

Using `as const` for now is, I guess, part of the cost of maintaining snowflakes like this.

Reply:
interesting.. lots of ways we could define colour variants like this.. will have a think 💭